grails-core/src/test/groovy/grails/web/CamelCaseUrlConverterSpec.groovy

```diff
@@ -28,6 +28,8 @@ class CamelCaseUrlConverterSpec extends Specification {
             'MyURLHelper'          | 'myURLHelper'
             'myURLHelper'          | 'myURLHelper'
             'MYUrlHelper'          | 'MYUrlHelper'
+            'myNamespace.v1'       | 'myNamespace.v1'
+            'MyNamespace.v1'       | 'MyNamespace.v1'
```

Comment:
Why shouldn't this be converted to `myNamespace.v1` ?

Reply:
Yes, I know it's strange, that's why I'm not convinced this is the best way to fix the issue. Maybe it's better just do `def namespace = controller.namespace` here https://github.com/grails/grails-core/blob/3.3.x/grails-web-url-mappings/src/main/groovy/org/grails/web/mapping/mvc/AbstractGrailsControllerUrlMappings.groovy#L133

What do you think @jameskleeh?

Reply:
I don't think the best solution is to not pass it through the converter. I think that would produce undesirable results.